pkg/sql/planner.go

```diff
@@ -473,6 +474,11 @@ func (p *planner) DistSQLPlanner() *DistSQLPlanner {
 	return p.extendedEvalCtx.DistSQLPlanner
 }
 
+// MigrationRunDependencies returns the migration.Cluster if there is one.
```

Comment:
Did we mean to say migration.Cluster here?

Reply:
Fixed. I was toying for a few minutes with what migrations will look like for tenants but backed off a bit.